code/modules/telesci/dses.dm

```diff
@@ -168,11 +168,16 @@
 		if(!detected)
 			if(user)
 				to_chat(user, "<span class = 'warning'>No structures detected.</span>")
-			playsound(src, 'sound/machines/buzz-sigh.ogg', 50, 1, gas_modified = 0)
+				user.playsound_local(get_turf(src), 'sound/machines/buzz-sigh.ogg', 50, 1, gas_modified = 0)
+			else
+				for(var/mob/M in get_turf(src))
+					M.playsound_local(get_turf(src), 'sound/machines/buzz-sigh.ogg', 50, 1, gas_modified = 0)
```

Comment:
This will play a sound emanating from the turf, which means it probably won't work in space assuming the space sound thing is fixed, so I'd keep that in mind.